packages/components/src/styles/components/side-nav/main.scss

```diff
@@ -111,6 +111,12 @@
     padding-right: var(--token-side-nav-wrapper-padding-horizontal-minimized);
     padding-left: var(--token-side-nav-wrapper-padding-horizontal-minimized);
   }
+
+  // Remove padding when it contains no child elements
+  // Note: styles will also apply if it contains only text nodes
+  &:not(:has(*)) {
```

Comment:
This is the solution I came up with so far to remove excess space when the SideNav header and footer aren't used (when it's paired with the `AppHeader`). Curious what others will think... 😅

Reply:
If you remove the whitespace using `~` in the Handlebars template and use `:empty` in CSS I _think_ it should work

Reply:
I kept trying that and there was still whitespace so I couldn't get it to work. I'll try again though.

Reply:
@KristinLBradley let me know if you want to pair on this (did you remove the whitespace in both the `index.hbs` and the `base.hbs` files?)

Reply:
Thanks, I looked at it again and realized I had missed adding "squishies" to the base.hbs component when I experimented previously. Once I added them there I was able to get it to work. :)